lib/_debugger.js

```diff
@@ -1557,7 +1556,16 @@ Interface.prototype.repl = function() {
 
     // Exit debug repl
     self.exitRepl();
-  });
+    
+    self.repl.rli.removeListener('SIGINT', exitDebugRepl);
+    self.repl.removeListener('exit', exitDebugRepl);
+  }
+  
+  // Exit debug repl on Ctrl + C
+  this.repl.rli.on('SIGINT', exitDebugRepl);
+  
+  // Exit debug repl on '.exit'
+  this.repl.on('exit', exitDebugRepl);
```

Comment:
It's still used in [L762](https://github.com/fb55/node-1/blob/78da3eab140660d6b40a84b7f0fa009378293ee9/lib/_debugger.js#L762), or am I missing something?